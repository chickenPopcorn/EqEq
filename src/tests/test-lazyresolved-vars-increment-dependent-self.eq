Foo = {
 b = { c + 3; }
}

Foo: find b {
  c = 3;     /* b == 6 */

  print("`b` is %0.f\n", b);
  b = b + 1; /* b == 7 */
  print("`b = b + 1` is %0.f\n", b);
  b = b + 3; /* b == 10 */
  print("`b = b + 3 is %0.f\n", b);
}
