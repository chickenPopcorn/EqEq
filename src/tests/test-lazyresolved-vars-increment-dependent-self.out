`b` is 6
`b = b + 1` is 7
`b = b + 3 is 10
